portlet-tck_3.0/pom.xml

```diff
@@ -216,12 +216,18 @@
             <version>4.11</version>
             <scope>test</scope>
          </dependency>
-         <dependency>
-            <groupId>org.seleniumhq.selenium</groupId>
-            <artifactId>selenium-java</artifactId>
-            <version>2.53.0</version>
-            <scope>test</scope>
-         </dependency>
+         <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-api</artifactId>
+            <version>3.11.0</version>
+            <scope>compile</scope>
```

Comment:
@vsingleton Just curious, why the change from `<scope>test</scope>` to `<scope>compile</scope>`? The only other file in this PR is TCKSimpleTestDriver.java which doesn't seem to benefit from this change?

Reply:
[ERROR] /Users/apple/Projects/apache.org/asf/portals-pluto/portlet-tck_3.0/driver/src/main/java/javax/portlet/tck/driver/TCKSimpleTestDriver.java:[40,27] package org.openqa.selenium does not exist